libr/core/canal.c

```diff
@@ -5495,6 +5495,13 @@ static bool is_noreturn_function(RCore *core, RAnalFunction *f) {
 			case R_ANAL_OP_TYPE_RET:
 				r_anal_op_free (op);
 				return false;
+			case R_ANAL_OP_TYPE_JMP: {
+				RAnalFunction *target_f = r_anal_get_fcn_in (core->anal, op->jump, 0);
+				if (!target_f || target_f->addr != f->addr) {
+					r_anal_op_free (op);
+					return false;
+				}
```

Comment:
uhm static analysis tools will fail here because of the missing break

Reply:
ok 5e8c6912fae61fb4072578f701b9ae5961e6335d